distribution/build.gradle

```diff
@@ -278,6 +279,31 @@ configure(subprojects.findAll { ['archives', 'packages'].contains(it.name) }) {
   /*****************************************************************************
    *             Properties to expand when copying packaging files             *
    *****************************************************************************/
+  configurations {
+    ['libs', 'libsPluginCli', 'libsKeystoreCli', 'libsSecurityCli'].each {
+      create(it) {
+        canBeConsumed = false
+        canBeResolved = true
+        attributes {
+          attribute(Category.CATEGORY_ATTRIBUTE, objects.named(Category, Category.LIBRARY))
+          attribute(Usage.USAGE_ATTRIBUTE, objects.named(Usage, Usage.JAVA_RUNTIME))
+          attribute(Bundling.BUNDLING_ATTRIBUTE, objects.named(Bundling, Bundling.EXTERNAL))
+        }
+      }
+    }
+  }
+
+  dependencies {
+    libs project(':server')
+    libs project(path: ':libs:elasticsearch-plugin-classloader', configuration: 'archives')
+    libs project(path: ':distribution:tools:java-version-checker', configuration: 'archives')
+    libs project(path: ':distribution:tools:launchers', configuration: 'archives')
```

Comment:
So the first two of these don't actually declare any dependencies, so it would be functionally equivalent to just use the default configuration, no?

As for the launcher project, it does declare dependencies, but it's expected to be packaged here in a common libs dir. @rjernst should the launcher deps be considered "provided"? In which case I think it might make sense to flag them `compileOnly` and again bring in the default configuration.

Reply:
I don't know about compileOnly, but maybe I'm misunderstanding what that means in this context. The launchers are needed at runtime. They are a bunch of the classes called by our cli scripts prior actually running the main cli class. IIRC they are not depended on by any cli classes at compile time.

Reply:
+1 on using the default. I made the change to compileOnly. that sounds reasonable and is also how things are done already for the keystone-cli compile dependencies.